ooiservices/app/main/annotation.py

```diff
@@ -15,103 +15,104 @@
 from datetime import datetime
 from dateutil.parser import parse as date_parse
 import sqlalchemy as sa
-
+from ooiservices.app.uframe.controller import split_stream_name
 import json
+import requests
+
+#List all annotations. build 6
+@api.route('/annotation/<string:instrument>/<string:stream>')
+def get_annotations(instrument,stream):
+    try:
+        if 'startdate' in request.args and 'enddate' in request.args:
```

Comment:
@FBRTMaka you will always need a start and end time. im going to change it to the other endpoint for annotations eventually, but we wanted to get it in.